webhook/env.go

```diff
@@ -29,6 +30,8 @@ const (
 	webhookNameEnvKey = "WEBHOOK_NAME"
 
 	secretNameEnvKey = "WEBHOOK_SECRET_NAME" //nolint:gosec // This is not a hardcoded credential
+
+	tlsMinVersionEnvKey = "TLS_MIN_VERSION"
```

Comment:
Let's be consistent here

```suggestion
	tlsMinVersionEnvKey = "WEBHOOK_TLS_MIN_VERSION"
```